examples/Lorenz/Lorenz_example.jl

```diff
@@ -0,0 +1,381 @@
+# Import modules
+include("GModel.jl") # Contains Lorenz 96 source code
+
+# Import modules
+using Distributions  # probability distributions and associated functions
+#using StatsBase
+using LinearAlgebra
+#using StatsPlots
+using GaussianProcesses
+using Plots
+using Random
+using JLD2
+
+# Import Calibrate-Emulate-Sample modules
+using CalibrateEmulateSample.EnsembleKalmanProcesses
+using CalibrateEmulateSample.GaussianProcessEmulator
+using CalibrateEmulateSample.MarkovChainMonteCarlo
+using CalibrateEmulateSample.Observations
+using CalibrateEmulateSample.Utilities
+using CalibrateEmulateSample.ParameterDistributionStorage
+
+rng_seed = 4137
+Random.seed!(rng_seed)
+
+# Output figure save directory
+figure_save_directory = "/home/mhowland/Codes/CESPlots/figures/lorenz_96_dynamic/debug/T_180/"
+data_save_directory = "/home/mhowland/Codes/CESPlots/data/lorenz_96_dynamic/debug/T_180/"
+
+# Governing settings
+# Characteristic time scale
+τc = 5. # days, prescribed by the L96 problem
+# Stationary or transient dynamics
+dynamics = 2 # Transient is 2
+# Lognormal prior or normal prior?
+log_normal = false # THIS ISN't CURRENTLY IMPLEMENTED
```

Comment:
do we want the option if not implemented? or would you prefer to remove it

Reply:
It could be something someone wants to implement later, and there are flags for plotting.